packages/components/src/components/Log/Log.scss

```diff
@@ -53,6 +53,31 @@ pre.tkn--log {
     right: 0;
   }
 
+  .bx--copy-btn.scroll-btn {
+    position: fixed;
+    width: 1.6rem;
+    height: 1.6rem;
+    min-height: 1.6rem; //to override Carbon's .bx--btn
+    right: var(--log-element-right);
+  }
+
+  .button-container {
+    position: absolute;
+    clip-path: inset(0); // ensure the children with position:fixed are not shown outside this element.
```

Comment:
We need to resolve the issue with the Carbon tooltip fin still showing:

![image](https://user-images.githubusercontent.com/2829095/126146424-1f6958bc-a01c-48a8-98a6-6bf04c4a4333.png)

Reply:
this fin is defined in `button::before`. I added in css `content: none;` as a pseudo-element to remove it but to do it needed to add an id to the `Button` component as I couldn't define `content: none;` with classes ( Carbon's css has many classes in the selector so it's practically impossible to  have a higher specificity specificity without an id and with classes/elements). The alternative would be to have in-line styling.

To have a truly unique id ( to account for the cases of several `Log` components simultaneously in the dom), we would need to pass the pipelineRunId, stepName and taskRun as props to the `Log` component to generate a unique id. But already the way I have defined the css custom properties as global properties assumes there is only one `Log` component at given moment in the dom.

So, if we want to account for multiple `Log` components simultaneously in the dom, I need to make these custom properties as element properties instead of global ( or use in-line styling) and pass the pipelineRunId, stepName and taskRun as props to the `Log` component ( to be able to remove the fin).

I believe the assumption of one `Log` component at given moment in dom is a safe assumption given that this feature is already flag-enabled.

Reply:
> I believe the assumption of one Log component at given moment in dom is a safe assumption given that this feature is already flag-enabled.

Yes this is safe. The flag defaults to false, and it's not something we'll be documenting as part of the public API of the component due to the specific requirements that must be met to have this (and the auto-scroll) work correctly. 👍